topics/ecology/tutorials/lotus2-identifying-fungi/tutorial.md

```diff
@@ -0,0 +1,462 @@
+>    {: .comment}
+>
+{: .hands_on}
+
+> <question-title>Examine the parameter options</question-title>
+>
+> 1. What other clustering algorithms can you use inside LotuS2?
+> 2. What other amplicon types can LotuS2 classify?
+>
+> > <solution-title></solution-title>
+> >
+> >1. In the "LotuS2" tool's parameter page, under *"Clustering Algorithms"*, click on the drop down. You should see these tools:
+> >    - DADA2
+> >    - swarm
+> >    - CD-HIT
+> >    - VSEARCH
+> >
+> >    SPUN uses VSEARCH for our {ITS2} amplicon for the identification of fungi from soils.
+> >2. In *"Other Taxonomy Options"*, click on the drop down under *"Amplicon type"*:
+> >    - Default
+> >    - LSU
+> >    - SSU
+> >    - ITS
+> >    - ITS1
+> >    - ITS2
+> >
+> >    Here, we were using general fungal primers that amplify the {ITS2} region. In another tutorial we will see how to use the {SSU} region for specific amplification of {AMF}
+> >
+> {: .solution}
+>
+{: .question}
+
+# Examine the outputs
+
+The LotuS2 Galaxy tool creates 6 output files that you should see in your history, with names like this (the exact number after each data might be different):
+1. LotuS2 on data 8, data 6, and others: main log file
+2. LotuS2 on data 8, data 6, and others: mapping file
+3. LotuS2 on data 8, data 6, and others: Newick-formatted phylogenetic trees between sequences
+4. LotuS2 on data 8, data 6, and others: FASTA-formatted extended OTU seed sequences
+5. LotuS2 on data 8, data 6, and others: OTU abundance matrix
+6. LotuS2 on data 8, data 6, and others: Complete LotuS2 output
+
+> <hands-on-title>Examine outputs</hands-on-title>
+> Click on the little {% icon galaxy-eye %} (eye) icon next to each output in the history to examine it. An explanation of each file is given below.
+>
+> 1. **main log file**: If we had run LotuS on the command line, the contents of the `output/LotuSLogS/LotuS_run.log` would be this main LotuS2 run log file.
+> 
+>     It has information on all the parameters actually passed to the tool, and information on the start time of each step in the program, plus the output of each step.
+>   
+>     You should always check this file first to see if LotuS2 completed correctly, and approximately how many reads were used/classified. If this number is much lower than what you expected, that might indicate a problem with the run and the parameters.
+> 
+> 2. **mapping file**: You can ignore this file as it is a repeat of the mapping tsv file that we used as an input.
+> 
+> 3. **Newick-formatted phylogenetic trees between sequences**: A phylogenetic tree created from the ITS2 sequences. If you want to see what the tree looks like, you can copy the contents of this file and paste it at this [online tree viewer](http://etetoolkit.org/)treeview/
```

Comment:
another update of the workflow could be to use this tool: https://usegalaxy.eu/?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fnewick_utils%2Fnewick_display%2F1.6%2Bgalaxy1&version=latest